src/services/loginMenu.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import LoginAuth from '../services/firebase';
```

Comment:
Where does `LoginAuth` come from?

Reply:
The firebase auth that is called firebase.js

Reply:
Hmm, do you mean `FirebaseAuth`? I don't see a `LoginAuth` in `firebase.js`

Reply:
just named firebase.js to FirebaseAuth.js and moved it to components